Common/Utility/ExternalModCallUtils.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Terraria;
+using Terraria.ModLoader;
+
+namespace TheBereftSouls.Common.Utility
+{
+    public class ExternalModCallUtils
+    {
+        public static int GetItemFromMod(Mod mod, string item)
+        {
+            return mod.Find<ModItem>(item).Type;
+        }
```

Comment:
`Mod.Find<T>()` can throw an exception<sup>\[[1]\]</sup>, wrapping it in a try-catch.

Also you can implement caching as that's also recommended in the docs<sup>\[[1]\]</sup>. You can use a simple `Dictionary<string, int>` *(or `Dictionary<(Mod, string), int>` for the off-chance of repeating item names, you can also just use a `(string, string)` instead where the first string is the mod name string literal and the second one is the item name)*. Here's some [technical documentation](https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2?view=net-8.0 "learn.microsoft.com") you can refer to for that.


[1]: https://docs.tmodloader.net/docs/stable/class_mod.html#a6e2234b93a8bdd9f6817e3e5eb7a7959 "tModLoader Docs"